libs/core/src/lib/list/directives/list-group-header.directive.spec.ts

```diff
@@ -39,6 +39,6 @@ describe('ListHeaderComponent', () => {
     });
 
     it('should assign class', () => {
-        expect(component.ref.nativeElement.className).toBe('fd-list__item fd-list__group-header');
+        expect(component.ref.nativeElement.className).toBe('fd-list__group-header');
```

Comment:
How trustworthy is this assertion?
There can be only 1 class name there?

Reply:
@aberikashvili there can be only 1 class name because I deleted class 'fd-list__item' as it was adding not nessasary styles that were described at the above comments.